### PR TITLE
BioEdit

### PR DESCRIPTION
**Pull request by GuangchuangYu**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [ ] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` reports no offenses.
- [ ] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask

### CODE REVIEW

File: Casks/bioedit.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+cask 'bioedit' do
+  version '7.1.9'
+  sha256 :no_check
+
+  ## info:
+  ## https://guangchuangyu.github.io/2016/03/yet-an-unofficial-bioedit-for-osx/
+  ##
+  url "https://gitlab.com/GuangchuangYu/BioEdit-OSX/raw/master/BioEdit.dmg"
```
Comment (miccal) [inline Casks/bioedit.rb]: This does not appear to be an official build - where did you find this `url`?

File: Casks/bioedit.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+cask 'bioedit' do
+  version '7.1.9'
+  sha256 :no_check
+
+  ## info:
+  ## https://guangchuangyu.github.io/2016/03/yet-an-unofficial-bioedit-for-osx/
```
Comment (neutric) [inline Casks/bioedit.rb]: I believe [unofficial builds are not accepted](https://github.com/caskroom/homebrew-cask/blob/7211b425e3db464210cf1d256710ef1965f615bf/doc/development/adding_a_cask.md#unofficial-vendorless-and-walled-builds). You might want to wait for feedback from the maintainers before putting more effort into this.

File: Casks/bioedit.rb (hunk 1 above)
Comment (GuangchuangYu) [inline Casks/bioedit.rb]: Yes, I build the wine version of this Windows software.

I will contact the maintainer to get his permission.

File: Casks/bioedit.rb (hunk 2 above)
Comment (vitorgalvao) [inline Casks/bioedit.rb]: @neutric is correct. Thank you, @GuangchuangYu, but we cannot accept this for security reasons. Our users needs to trust builds are officially recommended.

**Pull request closed without merging**